packages/slate-editor/gulpfile.js

```diff
@@ -61,7 +63,19 @@ function buildEsm(files = JS_DELIVERABLE_SOURCES) {
                     .pipe(gulp.dest('.css-modules/')),
             ]),
         )
-        .pipe(babel({ extends: './babel.config.json' }))
+        .pipe(
+            babel({
+                extends: './babel.config.json',
+                plugins: [
+                    [
+                        'transform-define',
+                        {
+                            'process.env.ENV_SLATE_VERSION': packageJson.version,
```

Comment:
@e1himself I just noticed that this is no accurate. We build slate before release and then this version will be outdated